index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>JB 45782 MISSION-1</title>
+    <script defer src="app.js"></script>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.7/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-LN+7fdVzj6u52u30Kp6M/trliBMCMKTyK833zpbD+pXdCLuTusPj697FH4R/5mcr" crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <h2>Add Product</h2>
+    <form id="product-form" onsubmit="addProduct(event)">
+        <label>Product Name</label><br>
+        <input id="product-name" type="text" required><br><br>
+        <label>Price</label><br>
+        <input id="product-price" type="number" required><br><br>
+        <label>Category</label><br>
+        <select id="product-category" required>
+            <option value="">Choose Category</option>
```

Comment:
should be `disabled` so the user cannot re-select it after a real value had been selected